### ISSUE TITLE
Interface bottom row overlaps and blocks UI

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#3590]
This is at 100% interface scale.
![image](https://user-images.githubusercontent.com/3536496/35253319-9fc58844-ff99-11e7-8586-13e39cbd9d40.png)

We'll need to make the interface not overlap for any selectable interface scale and resolution.  
One way to do this is to put all the bottom elements under a single control, with the empty sections implemented as spacers, and it scales the rest accordingly.

**Comment by MarjieAVolk** [comment]
I think it has always been like this, it is just a bit worse now because of the skills button moving.  Reducing the # of buttons on the action bar will help a lot.  Moving the skills button back to the other side would also help, if we decide we want to do that.

**Comment by MarjieAVolk** [comment]
I've improved this by removing some buttons from the action bar.  At the default interface scale, my monitor has tons of extra room between the elements.  (Default is 70% by the by.  I have confirmed this.  I can turn it up to ~90% before it starts to overlap.  Max on the slider is 125%).  There are lots of things that have problems at high scales, so I don't think this is a terrible issue.

This is what I think we should do to continue mitigating this issue:
- Decrease the default interface scale a bit
- Decrease the upper bound on interface scale.  Eg. instead of going up to 125% we can have it only go up to only 90%
- Change the label on the slider to show the default as being 100%.  Or show the max as being 100%, just something sane, the actual value doesn't matter to the player
- Possibly set a player's scale the first time they log in based on their monitor resolution
